src/main/java/com/passbook/sparkeighteen/service/TransactionService.java

```diff
@@ -19,13 +20,18 @@ public class TransactionService {
 
     private final UserRepository userRepository;
 
+    /**
+     * Instantiates a new Transaction service.
+     * @param transactionRepository the transaction repository
+     * @param userRepository        the user repository
+     */
     public TransactionService(TransactionRepository transactionRepository, UserRepository userRepository) {
         this.transactionRepository = transactionRepository;
         this.userRepository = userRepository;
     }
 
     /**
-     * Create method to do the actual transaction contaning the business logic to add deposit and withdraw and return response according
+     * Create method to do the actual transaction containing the business logic to add deposit and withdraw and return response according.
```

Comment:
next time please do changes related to your pr only.

Reply:
there was a spelling mistake so I changed, next time won't change anything unrelated to my PR.